clang/include/clang/Basic/LangOptions.h

```diff
@@ -1107,6 +1113,66 @@ inline void FPOptions::applyChanges(FPOptionsOverride FPO) {
   *this = FPO.applyOverrides(*this);
 }
 
+// The three atomic code-generation options.
+// The canonical (positive) names are:
+//   "remote_memory", "fine_grained_memory", and "ignore_denormal_mode".
+// In attribute or command-line parsing, a token prefixed with "no_" inverts its
+// value.
+enum class AtomicOptionKind {
+  RemoteMemory,       // true means remote memory is enabled.
```

Comment:
nit: just remove 'true means' from these three comments, it is implied.  Perhaps reform to:

`enable remote memory.`
`enable fine-grained memory.`
`ignore floating-point denormals.`

Reply:
will do